<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QMainWindow" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SignUp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FName_edit">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>161</y>
      <width>269</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="SName_edit">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>190</y>
      <width>269</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="AbYourSlf_edit">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>218</y>
      <width>269</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>125</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="FName_label">
       <property name="font">
        <font>
         <family>Old English Text MT</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>First Name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SName_label">
       <property name="font">
        <font>
         <family>Old English Text MT</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Second Name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="AbYourSlf_label">
       <property name="font">
        <font>
         <family>Old English Text MT</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>About Yourself :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="Login_label">
       <property name="font">
        <font>
         <family>Old English Text MT</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Login/Nickname* :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Pass_label">
       <property name="font">
        <font>
         <family>Old English Text MT</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Password* :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="Login_edit">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Pass_edit">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
